scripts/ci/find-github-pull-request

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+if [[ -z "$GITHUB_TOKEN" ]]; then
+	echo 'error: GitHub token is not set'
+	exit 255
+fi
+
+function github_api() {
+	response=$(mktemp)
+	response_code=$(curl --silent \
+		--header "Authorization: token $GITHUB_TOKEN" \
+		--output $response \
+		--write-out "%{http_code}" \
+		https://api.github.com/$1)
+
+	cat $response
+
+	if [[ $response_code -ne 200 ]]; then
+		exit 255
+	fi
+}
+
+user=${GITHUB_USER:-koding}
+repo=${GITHUB_REPO:-koding}
+head=${1:-$(git rev-parse HEAD)}
+
+response=$(github_api search/issues?q=$head+type:pr+repo:$user/$repo)
+pull_request_numbers=$(echo $response | jq '.items[].number')
```

Comment:
at this moment i dont know what for you are going to use this script but we might need to add `is:merged` or `merged` filter where we sometimes need to close a PR and open a new one with the same contents
also we might add `head:<branch name>` (sandox?) to make things faster